### PR TITLE
naive yeoman wrappers

### PR DESCRIPTION
**Pull request by kirillseva**
@robertzk @peterhurford

### CODE REVIEW

File: R/bootstrap.R [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#' Bootstrap a new syberia project
+#'
+#' @param name character. The name of the folder where the project should be created.
+#' @export
+syberia_new <- function(name) {
+  stopifnot(is.character(name) && length(name) <= 1)
```
Comment (kirillseva) [inline R/bootstrap.R]: Should probably add checks through ```system("which npm")``` and ```npm list -g yo --depth=0```

File: R/bootstrap.R [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    setwd(paste(wd, name, sep="/"))
+  }
+  system("yo syberia")
+}
+
+#' Bootstrap a new model for a syberia project
+#'
+#' @param name character. The name of the model.
+#' @export
+syberia_new_model <- function(name) {
```
Comment (kirillseva) [inline R/bootstrap.R]: and here syberia_root should be != NULL

File: R/bootstrap.R [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  stopifnot(is.character(name) && length(name) == 1)
+  if (is.null(syberia_root())) stop("You need to create a syberia project first. Check if you are in the right directory.")
+  system(paste("yo syberia:model", name))
+}
+
+#' Bootstrap a new classifier for a syberia project
+#'
+#' @param name character. The name of the classifierb
+#' @export
+syberia_new_classifier <- function(name) {
```
Comment (peterhurford) [inline R/bootstrap.R]: Can we dry these?

```R
syberia_new <- function(name, type) {
  stopifnot(is.character(name) && length(name) == 1)
  if (!identical(type, 'model') && !identical(type, 'classifier') stop('Syberia new can only create models and classifiers.')
  if (is.null(syberia_root())) stop("You need to create a syberia project first. Check if you are in the right directory.")
  system(paste0("yo syberia:", type, " ", name))
}
syberia_new_classifier <- function(name) syberia_new(name, type = 'classifier')
syberia_new_model <- function(name) syberia_new(name, type = 'model')

Comment (kirillseva) [inline R/bootstrap.R]: :+1:

**Comment by peterhurford** [comment]
It would be cool to get this out soon.

**Comment by robertzk** [comment]
Closing for now. We can revisit, but I would like to not introduce additional language toolchain dependencies if possible.

**Pull request closed without merging**